tailwind.config.js

```diff
@@ -7,12 +8,30 @@ module.exports = {
   ],
   theme: {
     extend: {
-      backgroundImage: {
-        'gradient-radial': 'radial-gradient(var(--tw-gradient-stops))',
-        'gradient-conic':
-          'conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))',
+      colors: {
+        'brand-blue': '#1DA1F2',
+        'brand-green': '#17BF63',
+        'header-bg': '#131313',
+      },
+      fontSize: {
+        xs: '.75rem',
+        sm: '.875rem',
+        tiny: '.875rem',
+        base: '1rem',
+        lg: '1.125rem',
+        xl: '1.25rem',
+        '2xl': '1.5rem',
```

Comment:
This can't be changed. it throws an error: 
Parsing error: Identifier directly after number. (26:9)eslint
An identifier or keyword cannot immediately follow a numeric literal.ts(1351)